CDP4Composition/Mvvm/Types/DisposableReactiveList.cs

```diff
@@ -0,0 +1,261 @@
+﻿// ------------------------------------------------------------------------------------------------
+// <copyright file="DisposableReactiveList.cs" company="RHEA System S.A.">
```

Comment:
There is no corresponding TestFixture for the `DisposableReactiveList`, please add it and also write a page on the IME wiki explaining the use and why this class is so important.

Please also explain when the `DisposableReactiveList` shall be used and when the regular `ReactiveList` shall be used.